Sources/ComplexModule/Documentation.docc/ComplexModule.md

```diff
@@ -0,0 +1,63 @@
+# ``ComplexModule``
+
+Types and operations for working with complex numbers.
+
+## Representation
+
+The `Complex` type is generic over an associated `RealType`; complex numbers
+are represented as two `RealType` values, the real and imaginary parts of the
+number.
+```
```

Comment:
For broader compatibility with various Markdown viewer implementations, would it be possible to add an extra line before and after such code blocks?